docs/src/developing/programming-model/accounts.md

```diff
@@ -52,10 +51,9 @@ runtime enforces that the account's data (the program) is immutable.
 
 ## Creating
 
-To create an account a client generates a _keypair_ and registers its public key
-using the `SystemProgram::CreateAccount` instruction with preallocated a fixed
-storage size in bytes. The current maximum size of an account's data is 10
-megabytes.
+To create an account, a client generates a _keypair_ and registers its public key
```

Comment:
How about just change "a" to "and" in "preallocated and fixed"?

Reply:
hi Jack...missed seeing your comment "How about just change "a" to "and" in "preallocated and fixed"...let me go back and look at that again

Reply:
Not sure it is relevant after the latest round of changes :-)